packages/server/src/module/Renderer/src/Renderer/Factory/RendererStorageFactory.php

```diff
@@ -51,9 +51,6 @@ public function createService(ServiceLocatorInterface $serviceLocator)
                 'serializer',
             ],
         ];
-        if (array_key_exists('markdown_cache', $serviceLocator->get('Config'))) {
```

Comment:
did you check if theres a production config for this?

Reply:
yep